### PR TITLE
Personal commands added

### PR DESCRIPTION
**Pull request by DUFCLiam**
Here are all the personal commands that are in the board: [Personal Commands](http://chicken-craft.boards.net/board/75/personal-commands)

### CODE REVIEW

**Comment by ThePheonixBlader** [comment]
@AvalancheYT

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java [hunk 1]
```diff
@@ -68,6 +68,69 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
+                    ItemStack cookie = new ItemStack(Material.CAKE, 1);
+                    ItemMeta meta = cake.getItemMeta();
+                    meta.setDisplayName(ChatColor.WHITE + "" + ChatColor.BOLD + "Death" + ChatColor.DARK_GRAY + "" + ChatColor.BOLD + "Cake");
+                    cake.setItemMeta(meta);
+                    player.getInventory().addItem(cake);
+                    player.sendMessage(ChatColor.AQUA + "Have some" + ChatColor.RED + "DEATH CAKE!");
+                }
+                break;
+            case "Burnsson":
+                public static final String CAKE_LYRICS = "There's no point crying over every mistake, sometimes you just have to admit that rare diamonds are awesome!";
```
Comment (AvalancheYT) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java]: To L100, copied codes... Please don't copy the codes.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java [hunk 2]
```diff
@@ -68,6 +68,69 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
+                    sword.addUnsafeEnchantment(Enchantment.FIRE_ASPECT, 100);
+                    ItemMeta meta = sword.getItemMeta();
+                    meta.setDisplayName(ChatColor.GOLD + "" + ChatColor.BOLD + "Legendary Fire Diamond");
+                    sword.setItemMeta(meta);
+                    player.getInventory().addItem(sword);
+                    FUtil.bcastMsg(output.toString());
+                }
+                break;
+                case "ThatOneMooShroom":
+                    public static final String TACO_LYRICS = "I Like Tacos! Here are some for you too!";
```
Comment (AvalancheYT) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java]: To L133, same as above.

**Comment by OxLemonxO** [comment]
@IllX why does it matter, this is just copied from /cake, so it doesn't matter

**Comment by DUFCLiam** [comment]
@AvalancheYT. May this be merged? The code copied will not breach copyright in the [TotalFreedom licence](https://github.com/TotalFreedom/License/blob/master/LICENSE.md)

**Comment by AvalancheYT** [comment]
I don't PRs but instead I add it to my code with other commitments.

**Comment by DUFCLiam** [comment]
But then the person that has done the code does not get credited. As you make a commit that only puts your status (Commits and Additions/deletions) So in there can me add a one line statement stating who wrote the code at the top?

@AvalancheYT

**Comment by AvalancheYT** [comment]
@DUFCLiam, I am the one who created the command.

**Comment by OxLemonxO** [comment]
IllX, why do you not merge PR's??

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java (hunk 1 above)
Comment (DUFCLiam) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java]: It is a StringBuilder and changes the color; We will get copyrighted for something built into java.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java (hunk 2 above)
Comment (DUFCLiam) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_personal.java]: It is a StringBuilder and changes the color; We will get copyrighted for something built into java.

**Pull request closed without merging**